### ISSUE TITLE
Simple pointing dropdown closes before trying to hover on the dropdown item

### ISSUE THREAD

**Issue by itsashis4u** [Semantic-Org/Semantic-UI#5322]

**Comment by Banandrew** [comment]
Hi, please provide a test case as per our [contributing guidelines](https://github.com/Semantic-Org/Semantic-UI/blob/master/CONTRIBUTING.md); you can fork [this JSFiddle](http://jsfiddle.net/efp8z6Ln).

**Comment by itsashis4u** [comment]
Please check the fiddle for example.
http://jsfiddle.net/3o5sx34f/1/

**Comment by ardamose123** [comment]
It seems this can be solved by using CSS's transition delay (https://css-tricks.com/almanac/properties/t/transition-delay/). I'll check this this evening.

**Comment by itsashis4u** [comment]
@ardamose123 I'm trying to do the same.

**Comment by itsashis4u** [comment]
@ardamose123 Any luck?

**Comment by ardamose123** [comment]
@itsashis4u Nope, transition-delay doesn't work as I expected.

**Comment by itsashis4u** [comment]
Any update on this?

**Comment by itsashis4u** [comment]
@Banandrew Any update?

**Comment by Banandrew** [comment]
@itsashis4u No. I can’t answer it, because I’m not maintaining the project.

**Comment by nbkhope** [comment]
This problem is driving me nuts :)

**Comment by lubber-de** [comment]
Fixed. See http://jsfiddle.net/pjga38wv/
